lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency.js

```diff
@@ -118,25 +114,6 @@ const expectations = {
         source: 'body',
       },
     ],
-    JsUsage: {
```

Comment:
The important part here is captured in `unused-javascript` below, especially the `/some-custom-url.js` bit.

Reply:
(note: splitting off the sourceUrl bit now)